src/main/java/com/spikes2212/command/drivetrains/commands/DriveTank.java

```diff
@@ -6,8 +6,10 @@
 import java.util.function.Supplier;
 
 /**
+ * @author
  * This command moves a {@link TankDrivetrain} using speeds supplied to the left and the right sides independently.
```

Comment:
`a command that moves` instead of `this command moves`

Reply:
a mistake